pkg/cmd/types.go

```diff
@@ -59,3 +65,44 @@ func (i *ImageInspectorOptions) Validate() error {
 	}
 	return nil
 }
+
+// OpenSCAPStatus is the status of openscap scan
+type OpenSCAPStatus string
+
+const (
+	StatusNotRequested OpenSCAPStatus = "NotRequested"
+	StatusSuccess      OpenSCAPStatus = "Success"
+	StatusError        OpenSCAPStatus = "Error"
+)
+
+type OpenSCAPMetadata struct {
+	Status           OpenSCAPStatus // Status of the OpenSCAP scan report
```

Comment:
should these be private since you're providing setter methods with special logic?  If so we'll want to provide accessors as well probably.